jobserver/templates/snapshot_detail.html

```diff
@@ -36,14 +36,20 @@
   </ol>
 </nav>
 
-<h1 class="h3">
-  {% if snapshot.is_draft %}
-  Outputs for {{ snapshot.workspace.name }}
-  <span class="badge badge-secondary">Draft</span>
-  {% else %}
-  Published outputs for {{ snapshot.workspace.name }}
-  {% endif %}
-</h1>
+<div class="d-flex justify-content-between align-items-center">
+  <h1 class="h3">
+    {% if snapshot.is_draft %}
+    Outputs for {{ snapshot.workspace.name }}
```

Comment:
I might be tempted to mirror this language, rather than just badge it. So either output

"Draft outputs for ..." or  "Published outputs for ..."

but just a suggestion

Reply:
Yeah, good call 👍